database/migrations/2021_09_23_221026_create_items_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->id();
             $table->foreignId('entry_id')->constrained();
             $table->foreignId('character_id')->constrained();
-            $table->string('name');
+            $table->text('name');
```

Comment:
This is probably fine, but keep in mind we also have the option to keep is as a string, but instead increase the max character count, something like: 
```suggestion
            $table->string('name', 500);
```
should do the trick. Mostly because `text(...)` fields are stored as files, so they have a higher i/o cost to retrieve :)

Reply:
ah okay ill use string in that case!